### PR TITLE
Simplify getting started guidance

### PR DESCRIPTION
**Pull request by mavam**
This PR simplifies the getting started instructions to make them more accessible for new users.

### CODE REVIEW

**Comment by mavam** [comment]
@dominiklohmann @dit7ya we should probably add more screenshots of the app. I would say:
1. A pipeline table (chart?)
2. A dashboard
3. A pipeline overview
WDYT?

**Comment by dit7ya** [comment]
Yes, fully onboard with this - just somebody needs to keep this updated.

File: README.md [hunk 1]
```diff
@@ -22,29 +22,31 @@ Open Source Data Pipelines for Security Teams
 
 ## Start Here
 
-Dive right in for free at [app.tenzir.com][app] and explore the cloud-based
-Tenzir demo. When you're ready for deploying your own node, run our installer
-that guides your through setup process:
+Dive right in at [app.tenzir.com][app] and play with a cloud-based Tenzir demo.
+Once you're ready for deploying your own node, run our installer that guides
+your through setup process:
```
Comment (dominiklohmann) [inline README.md:27]:
```suggestion
Dive right in at [app.tenzir.com][app] and play with a cloud-based Tenzir demo.
Once you're ready for deploying your own node, run our installer that guides
you through the setup process:
```

**Review by dominiklohmann — APPROVED** [review]

**Review by dominiklohmann — APPROVED** [review]
The screenshots are a really good addition. Thank you!

**Pull request merged**